src/openforms/js/components/admin/login.js

```diff
@@ -0,0 +1,9 @@
+const defaultLogin = document.querySelector('.admin-login-password');
+const loginForm = document.querySelector('#login-form');
+
+if (defaultLogin && loginForm) {
+  defaultLogin.addEventListener('click', e => {
+    loginForm.style.display = 'block';
+    defaultLogin.style.display = 'none';
+  });
+}
```

Comment:
I find this makes it hard to track where style rules come from, the preferred approach is to toggle a class name, or, with our BEM approach, a modifier. You can then fully control the hide/display logic in CSS, and we can also (optionally) incorporate it in Storybook should we wish to do so.

Reply:
you'll also want to put this code in `js/utils/dom::onLoaded` so that it doesn't matter if you load the script in the head tag or at the end of the body tag.